core/src/main/java/hudson/model/Action.java

```diff
@@ -90,15 +92,17 @@
      * @see Functions#isAnonymous()
      * @see Functions#getIconFilePath(Action)
      */
-    String getIconFileName();
+    @CheckForNull String getIconFileName();
```

Comment:
Just another case for code-style discussions. Oracle code style requires putting annotations on separate lines, but we have pretty much such code, because single-liners are being used by @jglick